save v16 minitree validation information

skalafut opened the pull request:
ran scripts/wrValidation.sh on partially complete v16 minitrees (missing DYAMC, DoubleEG RunD v4, and SingleMu RunD v3 and v4).  Logged event counts in minitrees, and events in Z mass window from DY tag and probe minitrees.

shervin86 commented on configs/miniTreeEntries.dat:
```diff
@@ -1,27 +1,90 @@
-DYToMuMu_powheg_6000toInf 17 12 2 51856 93227
-DYToMuMu_powheg_800to1400 7 17 0 65799 96024
+# DYJets_amctnlo_SHv16 dataset not found
+DYJets_madgraph 40 661932 1256 1635 3041984
```
Number of events increased in sidebands as expected by the bug fix. 
I don't remember if it should have an impact on the signal region as well.

skalafut replied:
Signal required !flavoursideband selection string, so I expected to see
more events in signal minitrees.


Regards,
Sean Kalafut

Physics PhD candidate
University of Minnesota Twin Cities

On Thu, May 12, 2016 at 4:56 PM, shervin86 <notifications@github.com> wrote:

> In configs/miniTreeEntries.dat
> <https://github.com/UMN-CMS/cms-WR/pull/50#discussion_r63034925>:
>
> > -DYToEE_powheg_50to120 6 8942 3 0 1250963
> > -DYToEE_powheg_6000toInf 14 3 50407 0 39798
> > -DYToEE_powheg_800to1400 2 25 50731 0 51549
> > -DYToEE_powheg 33 208811 22396 0 20598344
> > -DYToMuMu_powheg_120to200 2 17536 0 137 74462
> > -DYToMuMu_powheg_1400to2300 10 8 1 59718 95741
> > -DYToMuMu_powheg_200to400 8 523 0 38538 87023
> > -DYToMuMu_powheg_2300to3500 6 4 0 58958 95812
> > -DYToMuMu_powheg_3500to4500 8 3 0 58218 95189
> > -DYToMuMu_powheg_400to800 13 26 0 56216 94066
> > -DYToMuMu_powheg_4500to6000 6 8 0 56450 94687
> > -DYToMuMu_powheg_50to120 4 11714 0 2 1622657
> > -DYToMuMu_powheg_6000toInf 17 12 2 51856 93227
> > -DYToMuMu_powheg_800to1400 7 17 0 65799 96024
> > +# DYJets_amctnlo_SHv16 dataset not found
> > +DYJets_madgraph 40 661932 1256 1635 3041984
>
> Number of events increased in sidebands as expected by the bug fix.
> I don't remember if it should have an impact on the signal region as well.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UMN-CMS/cms-WR/pull/50/files/6f6e24e45f157b2a53ac61b7289cc99fe5b1ef40#r63034925>
>

shervin86 commented:
Now also SingleMu are done and powheg inclusive are finishing.
From the git commit reported in the files it seems that you have not used the master branch in order to produce the plots. Please submit the changes you made in one commit and the new values in a separate commit (same PR).

Commit:
removed underscore from analysis.cpp and updated scripts/wrValidation.sh

Commit:
updated ZPeakRegionIntegrals.txt and miniTreeEntries.dat in configs/ using complete v16 minitrees

skalafut commented:
updated scripts/wrValidation.sh, removed an underscore from analysis.cpp, and reran the validation script after all v16 minitrees were finished.  The numbers in configs/miniTreeEntries.dat and configs/ZPeakRegionIntegrals.txt reflect v16 minitrees.

shervin86 commented:
Can you double check if changes are already propagated to master? I think I have picked the important commits.

skalafut commented:
Yes the important changes have been propagated into master.


Regards,
Sean Kalafut

Physics PhD candidate
University of Minnesota Twin Cities

On Tue, May 17, 2016 at 9:32 AM, shervin86 <notifications@github.com> wrote:

> Can you double check if changes are already propagated to master? I think
> I have picked the important commits.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UMN-CMS/cms-WR/pull/50#issuecomment-219640867>
>

shervin86 commented:
Closing this PR
